.eslintrc.js

```diff
@@ -2,5 +2,13 @@ module.exports = {
   extends: 'frost-standard',
   globals: {
     capture: false
+  },
+  rules: {
```

Comment:
Why were these added?  Did you encounter any issues when you didn't have them?  (Wouldn't have expected you too since you've upgraded `ember-test-utils` to `^8.1.0`)

Reply:
Yes.
- Not sure on prroptypes, but complaining about Wrong Ones seems good
- `valid-test-descriptions` will fail if `it(...)` descriptions don't start with `'should ...'`.  This makes nearly every test in the repo fail linting.  We can either grandfather them in temporarily, or permanently -- I didn't see much to gain from rewriting all of the test descriptions.
- the no-multi-spaces one means you can put `  //` two spaces in front of a comment, which is nice but not necessary.

Reply:
The rules introduced by upgrading `ember-test-utils` to `^8.1.0` are the latest rules we are following as an organization.  As such your 2nd and 3rd bullet points need to be addressed, which means that the appropriate changes need to be made and not add additional repo-specific rules.  For the first bullet point if this is not already being provided by `ember-test-utils@^8.1.0` then an issue needs to be opened against it, as what is good for one repo is good for all repos.

Reply:
The tests for almost everything in this repo are written in a non-idiomatic style; is it OK to defer fixing those to a follow-up PR?  I'd like to have that be a separate PR both for legibility and because I had hoped to be able to use this feature this week. ;-)

Reply:
I'm not sure why you upgraded `ember-test-utils` as part of this PR but since you did then we should do the work that is involved with its upgrade as well.

Reply:
Besides, it's only Wednesday and the required changes shouldn't take more than 15-20 minutes tops ;-)